tests/php/Model/SiteTreeBacklinksTest.php

```diff
@@ -251,4 +264,24 @@ public function testLinkTrackingOnExtraContentFields()
         // assert backlink to page 2 no longer exists
         $this->assertNotContains($page2->ID, $page1->BackLinkTracking()->column('ID'), 'Assert backlink to page 2 has been removed');
     }
+
+    public function testLinkTrackingWithUntitledObjectsDisplaysAReadableIdentifier()
+    {
+        $referencingObject = new SiteTreeBacklinksTestContentObject();
+        $referencingObject->Content = '<p><a href="[sitetree_link,id=2]">Page 2</a></p>';
```

Comment:
```suggestion
        $page2ID = $this->objFromFixture('Page', 'page2')->ID;
        $referencingObject->Content = '<p><a href="[sitetree_link,id=' . $page2ID . ']">Page 2</a></p>';
```

Reply:
oh duh, of course! Same fix, different context - completely glossed over that! :roll_eyes:

Reply:
I've pushed the fix using a variable to avoid fetching the fixture twice, plus it allowed a squash :)
Thanks!